src/integrationTest/java/org/opensearch/security/http/LdapAuthenticationCacheTest.java

```diff
@@ -0,0 +1,187 @@
+/*
```

Comment:
these look unrelated? should this be a separate PR?

Reply:
These are directly related to the real problem that this PR addresses which is most relevant when using LDAP auth. See comment https://github.com/opensearch-project/security/pull/5325/files#r2086565354